u2f-gae-demo/README.md

```diff
@@ -0,0 +1,15 @@
+# U2F Demo Deployment
+
+To deploy https://crxjs-dot-u2fdemo.appspot.com and configure the appspot
+server to the U2F extension you have to:
+ (1) Set the version value to 'crxjs' in appengine-web.xml 
+    <version>crxjs</version>
+ (2) Set the extension id in u2f-api.js to 'kmendfapggjehodndflmmgagdbamhnfd'
+     u2f.EXTENSION_ID = 'kmendfapggjehodndflmmgagdbamhnfd';
+     
+To deploy https://noext-dot-u2fdemo.appspot.com and configure the appspot
```

Comment:
Nit: put the noext block first? It's also the default, so it should probably the first thing the reader sees.